### PR TITLE
Add personal task manager

### PR DESCRIPTION
**Pull request by MananSoni42**
Make changes that were requested

### CODE REVIEW

File: To-do-list/manan/to_do_list_tasks.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        wallpaper = Image.open(self.get_wallpaper()).convert('RGBA')
+        # add to-do list on top of the wallpaper
+        final_img = self.set_up(wallpaper)
+        final_img.save('to_do_list_wallpaper.png')
+        path = os.getcwd() + '/to_do_list_wallpaper.png'
+        # change wallpaper
+        os.system(f'/usr/bin/gsettings set org.gnome.desktop.background picture-uri {path}')
+        return path
+
+    # sets up the tasks on the current wallpaper
```
Comment (Pro-Panda) [inline To-do-list/manan/to_do_list_tasks.py]: Use docstrings to explain what a function does, not comments

File: To-do-list/manan/to_do_list_tasks.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    # default: to-do-list-data.json
+    # to load from a custom file: load(<name of file>.json)
+    def load(self, in_file='to-do-list-data.json', notif=False):
+        with open(in_file) as f:
+            d = json.load(f)
+            self.tasks = d["to-do-list"]
+
+            self.audio = d["audio"]
+            if not os.path.exists(self.audio):
+                print(f'File {self.audio} no longer Exists! Switching to default reminder Tone')
```
Comment (Pro-Panda) [inline To-do-list/manan/to_do_list_tasks.py]: Don't use `f'<string>'`. It's not a popular PEP style and is not supported by all versions. Either use`format()` or string concatenation

**Comment by MananSoni42** [comment]
Changed to f strings to normal strings ( used .format() )
added docstrings at relevant places
created function menu() called in to_do_list.py in main

**Comment by Pro-Panda** [comment]
LGTM ?

**Pull request merged**